docs/docs/getting_started.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Getting Started
+tags: []
+---
+
+## A Quick Start
+
+The easiest way to use Noir is through `nargo` (inspired by Rust's `cargo`). You can use `noirup` to quickly install `nargo`:
+
+```bash
+curl -L noirup.dev | bash
```

Comment:
Who currently owns this domain?

Reply:
I have it for the time being

Reply:
To minimize complications / risks, best to:

- First transfer ownership, or
- Revert to the GitHub hyperlink
 
before merging.

Reply:
Agreed, I'd like this to be an "officially owned" domain before we push people towards it.

Reply:
I'll transfer the ownership before merging so we don't have to iterate later

Reply:
This domain is still pointing at https://raw.githubusercontent.com/AztecProtocol/aztec-packages/refs/heads/zpedro/bbup_and_docs/barretenberg/bbup/install which now 404s

Reply:
Both `noirup` and `bbup` seems to work